dripline/extensions/thermo_fisher_endpoint.py

```diff
@@ -0,0 +1,82 @@
+from dripline.core import Entity, calibrate, ThrowReply
+
+import logging
+logger = logging.getLogger(__name__)
+
+__all__ = []
+__all__.append("ThermoFisherGetHexEntity")
+__all__.append("ThermoFisherGetEntity")
+
+
+class ThermoFisherGetHexEntity(Entity):
+    '''
+    A simple endpoint which stores a value but returns that value + a random offset
```

Comment:
Probably not an accurate description anymore

Reply:
updated